apps/site/redirects.json

```diff
@@ -106,7 +106,7 @@
     },
     {
       "source": "/documentation/:path*",
-      "destination": "/en/docs/:path*"
+      "destination": "/docs/:path*"
```

Comment:
It actually was an intentional design choice. In the past the website had /docs pages. But now these same pages are on learn, hence we have also redirects from some of these docs to /learn.